src/radical/pilot/raptor/worker_mpi.py

```diff
@@ -1017,7 +640,7 @@ def start(self):
         # the worker should be started before the managers as the manager
         # contacts the workers with queue endpoint information
         worker_ok = mt.Event()
-        worker    = self.get_rank_worker()
+        worker    = MPIWorkerRank
```

Comment:
isn't the idea was that `MPIWorker` can be inherited and a corresponding method `get_rank_worker` return a different worker-rank?

Reply:
Dang, I am so glad you are reviewing this - yes, that was the point!  Reverted.